migration/migrator/data/course_tables.sql

```diff
@@ -1389,7 +1389,8 @@ CREATE TABLE public.polls (
     release_date date NOT NULL,
     image_path text,
     question_type character varying(35) DEFAULT 'single-response-multiple-correct'::character varying,
-    release_histogram character varying(10) DEFAULT 'never'::character varying
+    release_histogram character varying(10) DEFAULT 'never'::character varying,
+    release_answer character varying(10) DEFAULT 'never'::character varying
```

Comment:
There doesn't seem to be a migration associated with this change.  You should make a migration, run the migration, and then dump the changes (which will make edits to this file).  You can read more about it here: https://submitty.org/developer/development_instructions/migrations